react-app/src/features/faq/SupportPage.tsx

```diff
@@ -0,0 +1,113 @@
+// import { useEffect, useState } from "react";
+
+import { SupportSubNavHeader } from "@/components";
+import LeftNavigation from "../../components/SupportPage/navigationBar";
+import { useEffect, useState } from "react";
+import { oneMACFAQContent, QuestionAnswer } from "./content/SupportMockContent";
+import ExpandCollapseBtn from "../../components/SupportPage/expandCollapseBtn";
+import { Accordion, AccordionItem, AccordionContent, AccordionTrigger } from "@/components";
+import { useParams } from "react-router";
+
+const FaqAccordion = ({ question }: { question: QuestionAnswer[] }) => {
+  return (
+    <>
+      {question.map(({ anchorText, answerJSX, question }) => (
+        <AccordionItem
+          value={anchorText}
+          id={`${anchorText}-support`}
+          data-testid={`${anchorText}-support`}
+          key={anchorText}
+          className="border-none my-6"
+        >
+          <AccordionTrigger showPlusMinus className="text-left font-bold bg-neutral-100 px-5">
+            {question}
+          </AccordionTrigger>
+          <AccordionContent className="bg-white pt-4">{answerJSX}</AccordionContent>
+        </AccordionItem>
+      ))}
+    </>
+  );
+};
+
+export const SupportPage = () => {
+  const { id } = useParams<{ id: string }>();
+  const [openAccordions, setOpenAccordions] = useState<string[]>([]);
+
+  const expandAll = () => {
+    const allIds = [];
+    oneMACFAQContent.flatMap(({ qanda }) => {
+      qanda.map(({ anchorText }) => allIds.push(anchorText));
+    });
```

Comment:
I know we spoke about this last time, but shouldn't you be able to assign the value directly to `allIds`?

```ts
const allIds = oneMACFAQContent.flatMap(({ qanda }) => qanda.map(({ anchorText }) => anchorText));
```